bin/ycsb

```diff
@@ -78,6 +78,7 @@ DATABASES = {
     "orientdb"     : "com.yahoo.ycsb.db.OrientDBClient",
     "redis"        : "com.yahoo.ycsb.db.RedisClient",
     "s3"           : "com.yahoo.ycsb.db.S3Client",
+    "solr"         : "com.yahoo.ycsb.db.SolrDBClient",
```

Comment:
Nit but should this be SolrClient? Solr to me isn't a DB per say and to stick with existing convention of NameClient.